constants.py

```diff
@@ -98,6 +98,13 @@ def error_string(error, logname, with_hd):
 logsfs_label_prefix = 'logs-'
 logsfs_label = logsfs_label_prefix + '%s'
 
+rootpart_label = "root"
+backuppart_label = "backup"
+lvmpart_label = "Local SR"
+bootpart_label = "EPT Boot"
```

Comment:
I think you mean ESP rather than EPT... and echoing the above comment, I'd suggest simply using `esp`.

Reply:
except it's an initialism for EFI System Partition so should stay in caps.